paywall/src/_server.js

```diff
@@ -19,9 +19,9 @@ function _server(port, dev) {
 
           // assigning `query` into the params means that we still
           // get the query string passed to our application
-          if (pathname.match('/0x')) {
+          if (pathname.match('/0x') || pathname === '/') {
             const params = route('/:lockAddress/:redirect?')(pathname)
-            app.render(req, res, '/', Object.assign(params, query))
+            app.render(req, res, '/paywallHome', Object.assign(params, query))
```

Comment:
This is a bit confusing. Does this mean the home page is at `/paywallHome`?

Reply:
see above.

Reply:
Well this is exactly what I was saying i my PR then ;) Next is too strict.
Of course we want the home page to be available on `/` and not expect the user to go to `/paywallHome`.